.buildkite/pipeline.yaml

```diff
@@ -2,13 +2,19 @@ $schema: 'http://json.schemastore.org/buildkite'
 steps:
   - label: ':docker:'
     command: .buildkite/validate-docker-compose.sh
+    agents:
+      queue: 'baremetal'
+    env:
+      PATH: "/var/lib/buildkite-agent/.asdf/shims:/var/lib/buildkite-agent/.asdf/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin"
```

Comment:
Can't we modify the path in the agent image instead? Seems odd to do this here

Reply:
the agent would not pick up the PATH defined on the physical server. It is defined as part of the build image, but even when checking the path as part of the PATH is not picked up.

Reply:
Oh interesting, I see

Reply:
note, this is all running as buildkite-agent at present, which I figured was a better approach than running as root.